oss.krtirtho.Spotube.yml

```diff
@@ -0,0 +1,51 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/master/data/flatpak-manifest.schema.json
+app-id: oss.krtirtho.Spotube
```

Comment:
I doubt this app-id is valid as `krtirtho.oss` isn't a valid domain

Reply:
It might be invalid. I didn't know it due to lack of knowledge about domains & I can't help it since its already deployed in other platforms too

Reply:
this is a blocker as it violate the spec. You can use a github based ID like `com.github.KRTirtho.Spotube`

Reply:
Well then I've to use a different appId for Flatpak. I'll rename the files & id with new `com.github.KRTirtho.Spotube` specifically for flathub
Thanks for helping me through this